examples/event-desc-multiple-descType.xml

```diff
@@ -3,20 +3,20 @@
     <div begin="10s" end="13s">
       <ttm:desc daptm:descType="scene">Scene 1</ttm:desc>
       <ttm:desc daptm:descType="plotSignificance">High</ttm:desc>
-      <p daptm:langSrc="original">
+      <p xml:lang="en">
         <span>A woman climbs into a small sailing boat.</span>
       </p>
-      <p xml:lang="fr" daptm:langSrc="translation">
+      <p xml:lang="fr" daptm:langSrc="en">
         <span>Une femme monte à bord d'un petit bateau à voile.</span>
       </p>
     </div>
     <div begin="18s" end="20s">
       <ttm:desc daptm:descType="scene">Scene 1</ttm:desc>
       <ttm:desc daptm:descType="plotSignificance">Low</ttm:desc>
-      <p daptm:langSrc="original">
+      <p>
```

Comment:
Should we add `xml:lang="en"`?

Reply:
Done in [01d26df](https://github.com/w3c/dapt/pull/179/commits/01d26dfa237f62aa96d98db149b0e3fe825bd630)